src/main/java/frc/robot/constants/RobotComponents.java

```diff
@@ -95,4 +95,17 @@
                 CAN.Swerve.REAR_RIGHT_DRIVE_MOTOR_ID, DRIVE_MOTOR_CONFIG.withPID(
                 LOCAL_SWERVE_MODULES.rearRightModuleConstants.driveCoefs));
     }
+
+    protected static class IntakeComponents {
+        public static TrigonTalonSRX MOTOR = SwerveComponents.FRONT_LEFT_ENCODER;
+    }
```

Comment:
shouldn't swerve take from intake because the intake motor config matters?

Reply:
idk what you mean please explane